docs/subnets/network-runner.md

```diff
@@ -28,7 +28,7 @@ To build and install the binary locally (requires `golang` to be installed. Chec
 
 ```bash
 cd ${HOME}/go/src/github.com/ava-labs/avalanche-network-runner
-go install -v ./cmd/avalanche-network-runner
+go install -v ./cmd
```

Comment:
could you pls instead refer to using the install.sh script, instead of `go install`, as using the script results in a bin that can be asked for versioning information (--version flag)
```
curl -sSfL https://raw.githubusercontent.com/ava-labs/avalanche-network-runner/main/scripts/install.sh | sh -s
```
for install under `./bin/` (need to change text afterwards)
or maybe better
```
curl -sSfL https://raw.githubusercontent.com/ava-labs/avalanche-network-runner/main/scripts/install.sh | sh -s -- -b $GOPATH/bin
```
so text afterwards does not need to be changed

Reply:
I noticed text before the command should be changed, probably:
- remove everything from Installation title until the command, instead write something like `To install the latest binary locally`, and probably a note that `GOPATH` should be defined
- put second curl command (one that writes into `$GOPATH/bin`), replacing both the cd and the go install

Reply:
done thanks @felipemadero